frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -68,12 +76,14 @@
     template(slot='footer' v-if='!isConfirmation && rule')
       .c-footer
         i.icon-vote-yea
-        i18n(html='According to your voting rules, <strong>{value} out of {total} members</strong> will have to agree with this.' :args='{value: rule.value, total: rule.total}')
+        span(v-html='footerMessage')
```

Comment:
You could localize the string using `i18n` using `v-if` and `v-else` instead of creating `footerMessage` computed property. Of course, using a computed property to generate the string isn't wrong (you just need to make sure you wrap the strings properly, see comment below). I'm just commenting here to make sure you know that there's also a pure-markup way to do this (in case you prefer that), i.e.:

```pug
i18n(v-if='groupShouldPropose' html='According to your voting rules, <strong>{value} out of {total} members</strong> will have to agree with this.' :args='rule')
i18n(v-else html='Your group has less than 3 members, so <strong>this change will be immediate</strong> (no voting required).')
```